octoprint_mrbeam/__init__.py

```diff
@@ -552,6 +554,15 @@ def on_settings_save(self, data):
                 )
             if "focusReminder" in data:
                 self._settings.set_boolean(["focusReminder"], data["focusReminder"])
+            if "gcodeAutoDeletion" in data:
+                self._settings.set_boolean(
+                    ["gcodeAutoDeletion"], data["gcodeAutoDeletion"]
+                )
```

Comment:
Is this necessary now? I thought `octoprint.plugin.SettingsPlugin.on_settings_save(self, data)` now took care of that.